### PR TITLE
Small remaining fixes on the CMSSW PR

### PR DESCRIPTION
**Pull request by VourMa**
With this PR I am dealing with:
- https://github.com/SegmentLinking/cmssw/pull/30#discussion_r1830145812
- https://github.com/SegmentLinking/cmssw/pull/30#discussion_r1830663590
- https://github.com/cms-sw/cmssw/pull/45117#discussion_r1750009570
- https://github.com/cms-sw/cmssw/pull/45117#discussion_r1750009731

### CODE REVIEW

File: RecoTracker/LST/interface/LSTOutput.h [hunk 1]
```diff
@@ -16,16 +18,14 @@ class LSTOutput {
         seedIdx_(std::move(seedIdx)),
         trackCandidateType_(std::move(trackCandidateType)) {}
 
-  enum LSTTCType { T5 = 4, pT3 = 5, pT5 = 7, pLS = 8 };
```
Comment (slava77) [inline RecoTracker/LST/interface/LSTOutput.h:19]: if you wanted to have fewer updates, just `using LSTTCType = lst::LSTObjType;` should be enough here

**Comment by slava77** [comment]
a 142pre3 variant of CMSSW apparently finished OK.
@VourMa would you like to stay as is or update as in https://github.com/SegmentLinking/cmssw/pull/119/files#r1832804345 ?

**Comment by VourMa** [comment]
This could be merged as well, if it looks ok.

**Comment by VourMa** [comment]
@slava77 Sorry for missing your comments before... I applied your suggestion in https://github.com/SegmentLinking/cmssw/pull/119/commits/3606c2f9670093f0845bec2223e770e0bd060995.

**Review by slava77 — APPROVED** [review]

**Pull request merged**